apps/trading/components/footer/footer.tsx

```diff
@@ -70,13 +79,19 @@ export const NodeUrl = ({ url }: NodeUrlProps) => {
 
 interface HealthIndicatorProps {
   blockDiff: number | null;
+  blockUpdateMsLatency: number;
 }
 
 // How many blocks behind the most advanced block that is
 // deemed acceptable for "Good" status
 const BLOCK_THRESHOLD = 3;
```

Comment:
suggestion (non blocking): I think we should move these consts to the use-node-health file so that all 'health' logic is kept together

Reply:
So I let myself for a small refactoring of use-node-health for close whole logic in one hook